components/automate-chef-io/content/docs/applications.md

```diff
@@ -0,0 +1,162 @@
+
+The individual service rows are sorted by their level of criticality, from the "most critical" status to the "least critical" status - "critical", "warning", "unknown", and "OK" respectively.
+Individual disconnected services display a broken chain icon with a note of when the last health check was received.
+The most critical issues are shown at the top of the list, allowing you to quickly determine the next steps for remediation.
+
+## Filtering
+
+### Search Bar
+
+To use the search bar, first select the search bar, and then select your search field type from the drop-down menu. Available search field types include _Origin_, _Service Name_, _Version_, _Channel_, _Application_, _Environment_, _Site_, _Build Timestamp_, and _Group Name_.
+
+After selecting your search field type, enter your specific search term, or select one from the auto-populated list of existing values. Applying the filter changes the Service Groups list to display matching results and updates health status display accordingly.
+
+![EAS Filtered Service Groups List](/images/docs/eas-filtered-service-groups.png)
+
+
+### Health Status Count Buttons
+
+Above the _Service Groups_ table, a header displays a total count and a health status count of the displayed service groups.
+
+The _Total_ count tabulates the number of all service groups. Health status count fields include _Critical_, _Warning_, _OK_, or _Unknown_.
+Select the buttons next to _Total_ to filter service groups by their statuses: _Critical_, _Warning_, _OK_, or _Unknown_.
+
+## Service Group Details Display
+
+Selecting a Service Group row populates the right-hand sidebar with information about each service in the Service Group. To filter the individual services in a service group by their health status, select the health status button of your choice at the top of the Services Details sidebar.
+
+The individual service rows are sorted by their level of status criticality, from most critical to least critical: _Critical_, _Warning_, _Unknown_, and _OK_.
+Individual disconnected services display a broken chain icon with a note of when the last health check was received.
+The most critical issues are shown at the top of the list, allowing you to quickly determine the next steps for remediation.
+
+
+## Lifecycle Management
+
+Disconnected services are defined services that have not reported back with a health check message for a user-defined period of time.
+
+Use the Chef Automate Service Groups API endpoint to manage the activity of periodic jobs, and to assess disconnection status and removal of services.
+
+### Periodic disconnection checks
+
+#### Health Checks
+Chef Automate assesses the status of Chef Habitat services based on the receipt of a health check message.
+If a node does not send a health check message to Chef Automate after a defined period of time, then Chef Automate marks that node as disconnected.
```

Comment:
maybe configurable period of time? instead of defined.

Reply:
This is the `threshold` setting?